app/controllers/support/courses_controller.rb

```diff
@@ -3,6 +3,9 @@ class CoursesController < SupportController
     def index
       @courses = provider.courses.order(:name).page(params[:page] || 1)
       render layout: "provider_record"
+    rescue ActiveRecord::RecordNotFound
```

Comment:
Do we need a token test?

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/2345/commits/d89be83b5f20606a93cb6a5318f8607b033b548a